config-templates/modules.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+return [
```

Comment:
This file returns all modules/bundles to be loaded. The kernel supports environments, so we can indicate if they should be loaded in all or just in a few of them.